src/measure_theory/measure/finite_measure_weak_convergence.lean

```diff
@@ -154,6 +165,168 @@ def coe_add_monoid_hom : finite_measure α →+ measure α :=
 instance {α : Type*} [measurable_space α] : module ℝ≥0 (finite_measure α) :=
 function.injective.module _ coe_add_monoid_hom finite_measure.coe_injective coe_smul
 
+variables [topological_space α]
+
+/-- The pairing of a finite (Borel) measure `μ` with a nonnegative bounded continuous
+function is obtained by (Lebesgue) integrating the (test) function against the measure.
+This is `finite_measure.test_against'`. -/
+abbreviation test_against_nn (μ : finite_measure α) (f : α →ᵇ ℝ≥0) : ℝ≥0 :=
+(∫⁻ x, f x ∂(μ : measure α)).to_nnreal
+
+-- I believe the formulation is generally useful, except maybe the exact form
+-- of the assumption `f_bdd`.
+-- Where to place?
+lemma _root_.is_finite_measure.lintegral_lt_top_of_bounded_to_ennreal {α : Type*}
+  [measurable_space α] (μ : measure α) [μ_fin : is_finite_measure μ]
+  {f : α → ℝ≥0∞} (f_bdd : ∃ c : ℝ≥0, ∀ x, f x ≤ c) : ∫⁻ x, f x ∂μ < ∞ :=
+begin
+  cases f_bdd with c hc,
+  apply lt_of_le_of_lt (@lintegral_mono _ _ μ _ _ hc),
+  rw lintegral_const,
+  exact ennreal.mul_lt_top ennreal.coe_lt_top.ne μ_fin.measure_univ_lt_top.ne,
+end
+
+-- Switched to using `nndist`, and believe these two could be useful... (1)
+-- Where to place?
+lemma _root_.nnreal.nndist_zero_eq_val (z : ℝ≥0) : nndist 0 z = z :=
+by { simp only [nnreal.nndist_eq, max_eq_right, zero_sub', sub_zero', zero_le'], }
+
+-- Switched to using `nndist`, and believe these two could be useful... (2)
+-- Where to place?
+lemma _root_.nnreal.nndist_zero_eq_val' (z : ℝ≥0) : nndist z 0 = z :=
+by { rw nndist_comm, exact nnreal.nndist_zero_eq_val z, }
```

Comment:
this one and the next one could probably be simp lemmas. They should be moved close to the definition of `nndist` on `nnreal`, I'd say.

Reply:
Thank you! I moved this and two other lemmas to `topology/metric/basic.lean`, where `nndist` is defined. It looks like the earliest possibility. I also made these two simp lemmas.

(And sorry for the delay on my part! Moving the lemmas broke my caches on this branch, but today I managed to get them working again.)